lib/email/receiver.rb

```diff
@@ -121,12 +121,12 @@ def find_existing_and_update_imap
       incoming_email
     end
 
-    def ensure_valid_address_lists
+    def self.ensure_valid_address_lists(mail)
```

Comment:
If that methods is changing `mail`, it should end with a `!` to indicate it has side effects